masonry/src/widget/button.rs

```diff
@@ -82,7 +82,8 @@ impl Widget for Button {
             PointerEvent::PointerDown(_, _) => {
                 if !ctx.is_disabled() {
                     ctx.capture_pointer();
-                    ctx.request_paint();
+                    // Changes in pointer capture impact appearance, but not accessibility node
```

Comment:
This is great and exactly what I was hoping for when I asked about adding some comments. Thanks!